lib/src/main/java/space/taran/arklib/Lib.kt

```diff
@@ -14,7 +14,7 @@ data class LinkData(
   }
 }
 
-typealias ResourceId = Long
+typealias ResourceId = String
```

Comment:
We should use pair of integers for better performance and convert to `String` only when we need to display or store the id. By the way, displayable format for `ResourceId` must be `<filesize>-<crc32>` (filesize first, then hyphen, then crc32), the same as in new index implementation: https://github.com/ARK-Builders/arklib/pull/29

Storages like `.ark/tags` or `.ark/meta` must use the same format for `ResourceId`.